api/apps/articles/routes.coffee

```diff
@@ -16,6 +16,7 @@ User = require '../users/model.coffee'
       'Pass published=true to only view published articles.')
 
   Article.where req.query, (err, results) ->
+    debug err if err
```

Comment:
Hmm yeah you're right that it should get handed in the error handler. So assuming then that any error from the API fetch/mongo stuff is handled, what else could cause the hanging `superagent` request? `superagent timeout 10000ms GET https://stagingwriter.artsy.net/api/articles`. It's always after this error (called from the client) that memory consumption gets piled up.